### PR TITLE
cli: fix preserving default mediation fees in cli

### PR DESCRIPTION
**Pull request by andrevmatos**
Fixes # 

**Short description**
Hotfix for CLI not preserving mediation fees

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1.
2.

### CODE REVIEW

File: raiden-cli/src/index.ts [hunk 1]
```diff
@@ -347,14 +346,8 @@ function constructMediationFeeConfiguration(
     };
   }
 
-  for (const [address, configurationOfToken] of Object.entries(mediationFeeConfiguration)) {
-    mediationFeeConfiguration[address] = {
-      flat: 0,
-      proportional: 4000,
-      imbalance: 3000,
```
Comment (weilbith) [inline raiden-cli/src/index.ts:354]: Please check the commit message for the reason why we need to keep this.

**Review by weilbith — CHANGES_REQUESTED** [review]

File: raiden-cli/src/index.ts (hunk 1 above)
Comment (weilbith) [reply raiden-cli/src/index.ts:354]: Got handled. 👍🏾

**Review by weilbith — APPROVED** [review]
Now its fine 👍🏾

**Comment by andrevmatos** [comment]
No need for CI since there's no test for cli yet.

**Pull request merged**